README.md

```diff
@@ -23,8 +23,8 @@ jobs:
           id: ${{ secrets.APP_ID }}
           private_key: ${{ secrets.PRIVATE_KEY }}
       - run: "echo installations: '${{ steps.stats.outputs.installations }}'"
-      - run: "echo suspended: '${{ steps.stats.outputs.suspendedInstallations }}'"
```

Comment:
Why did you remove this?

https://github.com/gr2m/app-stats-action/blob/1f49f3ba2e3ac8d66f7b5e26182d9d44ee3f053b/lib/app-stats.js#L33-L38